lib/screens/world_stats_screen.dart

```diff
@@ -184,7 +184,7 @@ class _WorldStatsScreenState extends State<WorldStatsScreen> {
             'Select Country',
             style: Theme.of(context)
                 .textTheme
-                .headline3!
+                .displaySmall!
```

Comment:
And finally, some changes to widget methods that had their packages changed due to the packages being obsolete or with updated methods.